bindings/pyroot/src/TPyROOTApplication.cxx

```diff
@@ -101,7 +101,7 @@ Bool_t PyROOT::TPyROOTApplication::CreatePyROOTApplication( Bool_t bLoadLibs )
       if ( argl && 0 < PyList_Size( argl ) ) argc = (int)PyList_GET_SIZE( argl );
       char** argv = new char*[ argc ];
       for ( int i = 1; i < argc; ++i ) {
-         char* argi = PyROOT_PyUnicode_AsString( PyList_GET_ITEM( argl, i ) );
+         char* argi = (char*) PyROOT_PyUnicode_AsString( PyList_GET_ITEM( argl, i ) );
```

Comment:
This was added recently to the 6 series via [this commit](https://github.com/root-project/root/commit/94412f7eab8d2cbdb4c83cd29215e7e2b185640f). I suggest to try a cherry pick, since it also includes other fixes for Python 3.7.

Reply:
@pcanal That change raises an error further down.

Reply:
@etejedor cherry-picking that one looks good.

Reply:
@MaxNoe good, can you then close this PR and open a new one with the cherry pick that supersedes it? Thank you.